bugzilla/commentsdisk.go

```diff
@@ -322,65 +323,72 @@ ScanHeader:
 		text := sr.Text()
 		switch {
 		case strings.HasPrefix(text, "Status: "):
-			parts := strings.SplitN(text, " ", 3)
+			fmt.Printf("%q", text)
+			parts := strings.SplitN(text, " ", 2)
 			if len(parts) < 2 || len(parts[1]) == 0 {
 				continue
 			}
-			bug.Info.Status = parts[1]
+			bug.Info.Status = strings.TrimSpace(parts[1])
 			if len(parts) > 2 {
```

Comment:
Now that you've reduced `SplitN` to 2, this is impossible, right?  I'm not clear on the context, but maybe the expectation is to fill `Resolution` another way?

Reply:
Changing it to split 2 was incorrect, I changed it back to 3 and added Resolution to the test case.